research-synthesis-requesters.md

```diff
@@ -0,0 +1,70 @@
+## Research Synthesis: Requesters
+
+We are conducting research to write recommendations for the site referenced in last summer’s [FOIA Improvement Act](https://www.justice.gov/oip/oip-summary-foia-improvement-act-2016) that “allows a member of the public to submit a request for records… to any agency from a single website.” At a minimum, our work requires researching how to send requests to agencies in a way that fits into their existing processes and doesn’t add a delay or otherwise burden the process.
+ 
+We’re especially interested in understanding how this can be done in a way that improves the system as a whole.
+ 
+### This document is NOT:
+* Finished or complete
+* A list of everything we plan to fix or address
+* Quantitative, statistically significant, or generalizable
+
+### This document IS:
+* Initial research that is broader than what we plan to work on
+* Imperfect and incomplete: please let us know what we’re missing!
+* Qualitative: this research is used to find patterns to further examine and test
+ 
+**We are sharing our initial research because we know it is imperfect and would greatly appreciate your feedback. Please [open an issue](https://github.com/18F/foia-recommendations/issues/new) in GitHub or email us at [National.FOIAPortal@usdoj.gov](mailto:National.FOIAPortal@usdoj.gov), and keep our [contributing policy](https://github.com/18F/foia-recommendations/blob/master/CONTRIBUTING.md) in mind.**
```

Comment:
The IS NOT, IS, and disclaimer boilerplate feel redundant given that a person reading is likely coming from the research-update.md that already lays things out. I get that we're trying to drive it home, but I might remove the sections here and include a link to them in the other doc "Learn more about how this document fits into our research process...".